packages/devtools_shared/lib/src/memory/event_sample.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Displays heap object allocation monitoring (in the VM).  The mllocation
```

Comment:
mllocation --> allocation.
Add a blank line after the first line to make a proper class level comment.
The word "displays" confuses me a bit as this is more model logic than view logic. Maybe, "Controls heap object allocation monitoring..."?

Reply:
Yeah display is the the end result the data is all stored with a HeapSample.

/// Monitor heap object allocations (in the VM).  The allocation monitor will
/// cause 'start' event exist in the HeapSample. Immediately afterwards a
/// 'continues' event is added on each subsequent timestamp tick (HeapSample)
/// until another monitor start event.  A 'reset' event stops the 'continues'
/// event for one timestamp tick with a 'reset' event. Immediately after the
/// reset event a 'continues' event will again appear in the HeapSample's
/// MemoryEventInfo - until a new monitor is started. One monitor exists per
/// VM connection.